src/app/public/modules/lookup/types/lookup-select-mode.ts

```diff
@@ -0,0 +1 @@
+export type SkyLookupSelectMode = 'single' | 'multiple';
```

Comment:
We've been following a pattern lately of declaring these as string enums for easier documentation. How would you feel about something like this?
```
export enum SkyLookupSelectMode {

  /**
   * description goes here...
   */
  single = 'single',

  /**
   * description goes here...
   */
  multiple = 'multiple'
  
 }
```

Reply:
I'm fine with that. That code pre-dated me taking it over :) Done